io/plugins/eu.esdihumboldt.hale.io.jdbc.test/src/eu/esdihumboldt/hale/io/jdbc/test/AbstractDBTest.java

```diff
@@ -185,6 +201,38 @@ protected Schema readSchema() throws Exception {
 	}
 
 	/**
+	 * @param map binding map
+	 * @param schema schema
+	 * @throws Exception exception
+	 */
+	protected void checkBidingAndSqlType(Schema schema, Map<String, Class<?>> map) throws Exception {
```

Comment:
Please fix the spelling of the method name.

For the binding map, please provide a more detailed documentation (what is mapped to what?), so users of this method will know exactly what they have to provide.